tests/tamr_client/fake.py

```diff
@@ -27,7 +27,16 @@ def _to_kwargs(fake):
         path = req.get("path")
         url = "http://localhost/api/versioned/v1/" + path
 
-    return dict(method=req["method"], url=url, **resp)
+    if "ndjson" in resp.keys():
+        ndjson = resp.pop("ndjson")
+        return dict(
+            method=req["method"],
+            url=url,
+            body="\n".join((dumps(line) for line in ndjson)),
+            **resp
+        )
+    else:
+        return dict(method=req["method"], url=url, **resp)
```

Comment:
You can simplify this logic by using the second argument of `pop`:

```suggestion
    ndjson = resp.pop("ndjson", None)
    if ndjson is not None:
        resp["body"] = "\n".join((dumps(line) for line in ndjson))

    return dict(method=req["method"], url=url, **resp)
```